backend/ObjectTypeSet.h

```diff
@@ -391,6 +391,13 @@ class ObjectTypeSet {
     return retVal;
   }
   
+  static ObjectTypeSet fromVector(std::vector<objectType> types) {
+    ObjectTypeSet retVal;
+    for (auto t: types) retVal.insert(t);
+    retVal.isBoxed = true;
```

Comment:
Skąd wiadomo że to ma być boxed? Z definicji?

Reply:
Tak, założyłem, że jeśli używam tej funkcji, to wektor na wejściu będzie miał przynajmniej 2 typy (bo dla jednego typu jest zwykły konstruktor). Skoro są wątpliwości, to dopiszę asserta.